docs/transactions/index.md

```diff
@@ -1,4 +1,46 @@
 ---
 sidebar_position: 1
 title: Transactions
----
\ No newline at end of file
+---
+
+Macrometa provides support for user-definable transactions within a region. _Transactions_ allow you to perform multi-document transactions with individual begin, commit, abort commands. They work similar to the BEGIN, COMMIT, and ROLLBACK operations in relational database systems.
+
+## How They Work
+
+Simply put, transactions are a way to aggregate, or group, multiple API calls. The process is as follows:
+
+### 1. Create a Transaction
+
+When you create the transaction, you specify which collections the transaction will write to.
+
+Once you create a transaction, you will be given a transaction identifier.
+
+### 2. Make API Calls
+
+Make API calls that include the transaction identifier to include them in the transaction. All actions will run, but they are not permanent until you commit the transaction.
+
+### 3. Commit or Abort Transaction
+
+- To make all changes made within the transaction permanent, commit the transaction.
+- To roll back the actions, perhaps because one of them failed, abort the transaction.
+
+## Supported Operations
+
+Supported operations include:
+
+- Read and write documents to collections.
+- Get the number of documents in a collection.
+- Get the number of key-value pairs in a collection.
+- Truncate collections.
+- Run queries or query workers.
```

Comment:
BTW I just recalled that it is a known bug that transactions are not working with query workers yet, and it likely will not get fixed on time for 0.18.0.